ci/actions/install.sh

```diff
@@ -7,7 +7,4 @@ conda update -q conda
 conda info -a
 conda devenv
 source activate autodiff
-mkdir .build
-cd .build || exit
-cmake .. -GNinja
-ninja
+inv compile -c -n 2
```

Comment:
Hi @volpatto , all looks good! Thanks a lot for this. I would prefer, however, to have a simpler solution for compiling and running tests in Windows. The `tasks.py` file below would have to be dropped (I did not understand why it is necessary - it is quite long in comparison with the ninja commands above, and it would be great if we keep only CMake to managing the build tasks for the CI). Do we really need `tasks.py`?

Reply:
Thanks, @allanleal! I will tell you my opinion on each point.

IMHO, I think that this is why `invoke` is an interesting tool. We have the same command for each platform, the developer does not have to be concerned about several commands to use a build system. From the developer's point of view, compilation is `inv compile` and running the tests is `inv tests`. He/she does not have to be concerned about where is the test executable, neither the options to a pretty display of tests in the terminal. `inv` is a way to turn the develop mode the same for all platforms. So for me, it is simpler this way, because you hide the details of build system commands and use the same despite the OS. But, of course, this is my opinion and you are the core developer (and main developer). If there is no point to you, it's OK! We drop it, no worries.

> I did not understand why it is necessary - it is quite long in comparison with the ninja commands above, and it would be great if we keep only CMake to managing the build tasks for the CI

Actually, it is not _necessary_, but I think it can help. You can use `inv compile` once (say, in the first time you create your fork), to create the build for you, and then use `ninja` thereafter, no problems. Or you can do as implemented before, with commands to create a dir, to generate files with CMake and build. Again, I prefer the way to use `inv compile` encapsulating everything than running more commands, sounds more error-prone to me. But, again, if it's not useful for your workflow, I think that I should drop it, and there is no problem for me :) And BTW, CMake is managing the build tasks, `inv` encapsulates the CMake commands: for instance, `inv compile -c -n 4` is a clean (c) CMake build and install using 2 processes (n).

About everything: my vision is biased because I'm used to this workflow. I'm an occasional `autodiff` dev, you are a frequent dev, so please consider if these changes here are useful for you. I will comment below about what each task below.